content/_index.html

```diff
@@ -0,0 +1,495 @@
+---
+title: RAPIDS.ai | GPU accelerated data science
+description: homepage
+---
+
+<!-- Hero Shortcode -->
+{{< hero hero-text="GPU Accelerated Data Science" button-link="#quick-start" button-text="Quick Start">}}
+
+  <!-- Announcements Shortcode (0-3x)-->
+  <!-- UPDATE HERE -->
+  {{< announcements 
```

Comment:
I think the shadows on these announcement boxes are a little aggressive. Can we reduce the size of the shadow a bit?

I didn't see where this is defined in the source, but I can tell you what settings I like. I adapted the offsets of each shadow layer from `3px 5px` to `1px 2px` and from `6px 10px` to `3px 5px`. It looks tighter while retaining a subtle pop from the otherwise-flat site look.

```css
button {
  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05), 0 1px 2px rgba(0, 0, 0, 0.07), 0 3px 5px rgba(0, 0, 0, 0.1), 0 1px 2px rgba(0, 0, 0, 0.2) inset;
}
```

Before:
![image](https://user-images.githubusercontent.com/3943761/221244356-39bc4954-cc1b-4f26-9446-250cc53ee5a4.png)

After:
![image](https://user-images.githubusercontent.com/3943761/221244493-47e4c652-103e-44c4-9474-1fcf6322f6ea.png)